lib/Screens/Components/bottom_buttons.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:untitled3/Screens/Menu/main_menu_screen.dart';
+
+class BottomButtons extends StatelessWidget {
+  const BottomButtons({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Colors.blue,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        children: <Widget>[
+          Expanded(
+            child: Container(
+              color: Colors.blue[700],
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      TextButton(
+                        style: TextButton.styleFrom(
+                          primary: Colors.white,
+                        ),
+                        child: Column(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: [
+                            FaIcon(FontAwesomeIcons.home),
+                            Text('MENU'),
+                          ],
+                        ),
+                        onPressed: () {
+                          Navigator.push(
+                            context,
+                            MaterialPageRoute(
+                              builder: (context) => MainMenuScreen(),
```

Comment:
It seems to create another instance of the MainMenuScreen() object when onPressed: () is executed. I tested this by pressing Menu a bunch and then there were multiple "back" navigation arrows in the top left hand corner